src/Assets/DesktopTestProjects/MultiTFMXunitProject/XUnitProject/XUnitProject.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), testAsset.props))\testAsset.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net461;$(CurrentTargetFramework)</TargetFrameworks>
```

Comment:
I am torn about the naming here (and all other instances). This project now forces me to double check what CurrentTargetFramework is to understand what is going on. But I guess it is better at communicating the intent than having the variable named `net6` and then having to rename it every time there is an upgrade.